dts/arm/st/h7/stm32h743.dtsi

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright (c) 2020 Teslabs Engineering S.L.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <st/h7/stm32h7.dtsi>
+
+/ {
+	/* system data RAM accessible over over AXI bus */
+	sram0: memory@24000000 {
+		compatible = "mmio-sram";
+		reg = <0x24000000 DT_SIZE_K(512)>;
```

Comment:
Please, clarify here (for me at least): you specify 512kB for sram0, and sram0 is chosen as the image SRAM, however the .yaml specifies 1024kB SRAM. Am I missing something?

Reply:
These MCU series advertise 1024KB of SRAM, which includes AXI bus SRAM, DCM, backup, etc. This is why `sram0` is not 1024KB. Details can be found in RM0433 (s. 2.4, p. 133).

Reply:
OK so the Zephyr image SRAM is _correctly_ set to 512KB?

If so, then you need to correct the .yaml which shows a RAM of 1024kB. Note that _that_ value is used to filter samples & tests for this board, so it needs to be correct.

Reply:
Ok, fixed the yaml file :-)